source/dem/pp_nonlinear_force.cc

```diff
@@ -262,44 +264,48 @@ PPNonLinearForce<dim>::calculate_nonlinear_contact_force_and_torque(
   double normal_spring_constant =
     1.3333 * effective_youngs_modulus * radius_times_overlap_sqrt;
   double normal_damping_constant =
-    -1.8257 * model_parameter_beta * sqrt(model_parameter_sn * effective_mass);
+    -1.8257 * model_parameter_betha * sqrt(model_parameter_sn * effective_mass);
```

Comment:
```suggestion
    -1.8257 * model_parameter_beta * sqrt(model_parameter_sn * effective_mass);
```
Beta does not take an h